lib/ui/form/widget/form_survey_question_page.dart

```diff
@@ -69,71 +69,108 @@ class FormSurveyQuestionPage extends StatelessWidget {
     final displayType = question.displayType;
     switch (displayType) {
       case DisplayType.smiley:
-        return FormSurveyAnswerSmiley(
-          answers: question.answers,
-          onUpdateAnswer: (selectedAnswer) {
-            onUpdatedAnswers([selectedAnswer]);
-          },
-        );
+        if (question.answers.isNotEmpty) {
+          return FormSurveyAnswerSmiley(
+            answers: question.answers,
+            onUpdateAnswer: (selectedAnswer) {
+              onUpdatedAnswers([selectedAnswer]);
+            },
+          );
+        } else {
+          return const SizedBox.shrink();
+        }
       case DisplayType.nps:
-        return FormSurveyAnswerNps(
-          answers: question.answers,
-          onUpdateAnswer: (selectedAnswer) {
-            onUpdatedAnswers([selectedAnswer]);
-          },
-        );
+        if (question.answers.isNotEmpty) {
+          return FormSurveyAnswerNps(
+            answers: question.answers,
+            onUpdateAnswer: (selectedAnswer) {
+              onUpdatedAnswers([selectedAnswer]);
+            },
+          );
+        } else {
+          return const SizedBox.shrink();
+        }
       case DisplayType.heart:
-        return FormSurveyAnswerEmoji(
-          emoji: '❤',
-          answers: question.answers,
-          onUpdateAnswer: (selectedAnswer) {
-            onUpdatedAnswers([selectedAnswer]);
-          },
-        );
+        if (question.answers.isNotEmpty) {
+          return FormSurveyAnswerEmoji(
+            emoji: '❤',
+            answers: question.answers,
+            onUpdateAnswer: (selectedAnswer) {
+              onUpdatedAnswers([selectedAnswer]);
+            },
+          );
+        } else {
+          return const SizedBox.shrink();
+        }
       case DisplayType.star:
-        return FormSurveyAnswerEmoji(
-          emoji: '⭐',
-          answers: question.answers,
-          onUpdateAnswer: (selectedAnswer) {
-            onUpdatedAnswers([selectedAnswer]);
-          },
-        );
+        if (question.answers.isNotEmpty) {
+          return FormSurveyAnswerEmoji(
+            emoji: '⭐',
+            answers: question.answers,
+            onUpdateAnswer: (selectedAnswer) {
+              onUpdatedAnswers([selectedAnswer]);
+            },
+          );
+        } else {
+          return const SizedBox.shrink();
+        }
       case DisplayType.thumbs:
-        return FormSurveyAnswerEmoji(
-          emoji: '👍🏻',
-          answers: question.answers,
-          onUpdateAnswer: (selectedAnswer) {
-            onUpdatedAnswers([selectedAnswer]);
-          },
-        );
+        if (question.answers.isNotEmpty) {
+          return FormSurveyAnswerEmoji(
+            emoji: '👍🏻',
+            answers: question.answers,
+            onUpdateAnswer: (selectedAnswer) {
+              onUpdatedAnswers([selectedAnswer]);
+            },
+          );
+        } else {
+          return const SizedBox.shrink();
+        }
```

Comment:
I'm curious why do we need to add the `isNotEmpty` check to other widgets that don't have any `first` or single element usage. Actually, I only requested for `textarea` 🤔

Reply:
I thought this can be applicable for all, as the answer list is empty we don't need to show the ui, what do you think?

Reply:
@Shayokh144 Actually, if you want to apply for all cases, you can check it outside of `switch (displayType) {`

Reply:
@luongvo thanks for the idea.. applied